binding/SkiaSharp/SKRoundRect.cs

```diff
@@ -122,6 +122,12 @@ public void SetRectRadii (SKRect rect, SKPoint[] radii)
 		{
 			if (radii == null)
 				throw new ArgumentNullException (nameof (radii));
+
+			SetRectRadii(rect, radii.AsSpan());
+		}
+
+		public void SetRectRadii (SKRect rect, Span<SKPoint> radii)
```

Comment:
This might be better as a read-only span since the method does not edit the contents?

```suggestion
		public void SetRectRadii (SKRect rect, ReadOnlySpan<SKPoint> radii)
```